eng/performance/benchmark_jobs.yml

```diff
@@ -46,6 +46,8 @@ jobs:
             value: 'chmod +x ./tools/machine-setup.sh;. ./tools/machine-setup.sh'
           - name: PreservePythonPath
             value: 'export ORIGPYPATH=$PYTHONPATH'
+          - name: SetAllowOpenSsl102
+            value: 'export CRYPTOGRAPHY_ALLOW_OPENSSL_102 =true'
```

Comment:
This is not sufficient. Notice how we then use variables like `PreservePythonPath` when building the command line down on line 92. You need to include it there as well.